### PR TITLE
br: backup-log: Test and debug for backup-log process

### PR DESCRIPTION
**Pull request by joccau**
### What is changed and how it works?

Close https://github.com/tikv/tikv/issues/11679

What's Changed:
This pr is for test and debug for the backup-log flow

### Release note 

```release-note
None
```

### CODE REVIEW

File: br/cmd/br/stream.go [hunk 1]
```diff
@@ -108,6 +109,21 @@ func newSteramResumeComamnd() *cobra.Command {
 	return command
 }
 
+func newStreamStatusComamnd() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "status",
+		Short: "status a stream task",
```
Comment (kennytm) [inline br/cmd/br/stream.go:115]:
```suggestion
		Short: "get status of a stream task",
```

status is not a verb

File: br/cmd/br/stream.go [hunk 2]
```diff
@@ -108,6 +109,21 @@ func newSteramResumeComamnd() *cobra.Command {
...
+		Args:  cobra.NoArgs,
+		RunE: func(command *cobra.Command, _ []string) error {
+			// empty db/table means full backup.
```
Comment (kennytm) [inline br/cmd/br/stream.go:118]:
```suggestion
```

**Review by 3pointer — APPROVED** [review]
LGTM

File: br/pkg/stream/models.go [hunk 3]
```diff
@@ -44,7 +46,8 @@ func RangesOf(name string) string {
 // Normally it would be <prefix>/ranges/<task-name(string)>/<start-key(binary)> -> <end-key(binary)>
 func RangeKeyOf(name string, startKey []byte) string {
 	// We cannot use path.Join if the start key contains patterns like [0x2f, 0x2f](//).
-	return RangesOf(name) + string(startKey)
+	//return RangesOf(name) + string(startKey)
+	return path.Join(RangesOf(name), string(startKey))
```
Comment (YuJuncen) [inline br/pkg/stream/models.go:50]: Please either use the origin version `RangesOf(name) + string(startKey)` or remove the comment `// We cannot use path.Join if the start key contains patterns like [0x2f, 0x2f](//).`

**Review by YuJuncen — APPROVED** [review]
LGTM

**Pull request merged**